packages/server/lib/clients/auth.client.ts

```diff
@@ -32,7 +32,9 @@ export function setupAuth(app: express.Router) {
             store: sessionStore,
             name: 'nango_session',
             unset: 'destroy',
-            cookie: { maxAge: 7 * 24 * 60 * 60 * 1000, secure: true, path: '/', sameSite: 'none', httpOnly: true },
+            cookie: baseUrl.startsWith('https')
+                ? { maxAge: 7 * 24 * 60 * 60 * 1000, secure: true, path: '/', sameSite: 'none', httpOnly: true }
+                : { maxAge: 7 * 24 * 60 * 60 * 1000, secure: false, path: '/', httpOnly: true },
```

Comment:
what happened locally before when secure was set to true?

Reply:
before it was set to false actually